Makefile

```diff
@@ -325,6 +325,17 @@ check-ingress-conformance: | install-contour-working run-ingress-conformance cle
 run-ingress-conformance:
 	./test/scripts/run-ingress-conformance.sh
 
+.PHONY: deploy-gcp-bench-cluster
+deploy-gcp-bench-cluster:
+	./test/scripts/gcp-bench-cluster.sh deploy
+
+.PHONY: run-bench
+run-bench:
+	ginkgo -tags=e2e -mod=readonly -keep-going -randomize-suites -randomize-all -slow-spec-threshold=4h -timeout=5h -r -v ./test/e2e/bench
+
+.PHONY: bench
+bench: deploy-gcp-bench-cluster run-bench
```

Comment:
do we want to delete the cluster after running the test?

Reply:
added that, though with the makefile mechanics if youre relying on a generated name its sorta broken, you can pass a cluster name via env var though that should work